src/idsk/components/stepper/stepper.js

```diff
@@ -0,0 +1,288 @@
+
+/*
+  Stepper
+
+  This allows a collection of sections to be collapsed by default,
+  showing only their headers. Sections can be exanded or collapsed
+  individually by clicking their headers. An "Zobraziť všetko" button is
+  also added to the top of the accordion, which switches to "Zatvoriť všetko"
+  when all the sections are expanded.
+
+  The state of each section is saved to the DOM via the `aria-expanded`
+  attribute, which also provides accessibility.
+
+*/
+
+import { nodeListForEach, toggleClass } from '../../common'
+import '../../../govuk/vendor/polyfills/Function/prototype/bind'
+import '../../../govuk/vendor/polyfills/Element/prototype/classList'
+
+function Stepper($module) {
+  this.$module = $module
+  this.moduleId = $module.getAttribute('id')
```

Comment:
mali by sme zachovavat naming conventions, t.j. pouzivat `$` nebudem to pisat vsade ale dole to tiez absentuje tento znak

Reply:
zmenil si to vsade ale tu nie :))